mftf/2.3/suite.md

```diff
@@ -231,9 +243,10 @@ It may contain test steps with any [actions] and [action groups].
 A set of filters to be used for including tests into the suite.
```

Comment:
A set of filters that you can use to specify which tests to include in the suite

Reply:
:ok: